node_modules/oae-admin/tenants/bundles/default.properties

```diff
@@ -42,8 +46,16 @@ TENANTS_STOPPED_SUCCESSFULLY = All tenants where successfully stopped.
 TENANT_COULD_NOT_BE_STARTED = The tenant could not be started.
 TENANT_COULD_NOT_BE_STOPPED = The tenant could not be stopped.
 TENANT_CREATED = Tenant created.
+TENANT_EMAIL_DOMAIN_FAILED = Tenant email domain failed.
```

Comment:
Can you be more specific about when we use this pattern? Because our message keys seem to be predominantly underscore-separated versions of the english translation.